### DISCUSSION TITLE
fastapi cli command is not found

### DISCUSSION THREAD

**Discussion by Gelembjuk** [fastapi/fastapi#12248] [Questions]
### First Check

- [X] I added a very descriptive title here.
- [X] I used the GitHub search to find a similar question and didn't find it.
- [X] I searched the FastAPI documentation, with the integrated search.
- [X] I already searched in Google "How to X in FastAPI" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [X] I already checked if it is not related to FastAPI but to [Pydantic](https://github.com/pydantic/pydantic).
- [X] I already checked if it is not related to FastAPI but to [Swagger UI](https://github.com/swagger-api/swagger-ui).
- [X] I already checked if it is not related to FastAPI but to [ReDoc](https://github.com/Redocly/redoc).

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```python
[build@localhost ~]$ fastapi
-bash: fastapi: command not found
```

### Description

I have installed FastAPI using a totorial.
But FastAPI client is not found.

[root@roman-v3 ~]# python3
Python 3.6.8 (default, Mar 13 2024, 18:04:07) 
[GCC 4.8.5 20150623 (Red Hat 4.8.5-44)] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> 

The history of my attempts to install. After each command i have checked if fastapi command exists.
After most of install request i have reloaded terminal session (in case if path to fastapi is in env etc)

My pip3 had been upgraded to latest version 

```
# pip3 install "fastapi[standard]"
# pip3 install --only-binary :all: fastapi[all]
# pip3 install fastapi-cli
# pip3 install uvicorn
# python3 -m pip install fastapi uvicorn[standard]
# pip3 install "fastapi-cli[standard]"
```

Also i have repeated this in vitrual env too. Same result.
Also i have tried to two different servers

```
pip3 list
fastapi            0.58.1
fastapi-cli        0.0.1
pip                21.3.1
uvicorn            0.16.0 
```

What am i doing wrong?

### Operating System

Linux

### Operating System Details

Centos 7

### FastAPI Version

0.58.1

### Pydantic Version

1.9.2

### Python Version

Python 3.6.8

**Comment by sinisaos** [comment]
@Gelembjuk To use FastAPI, you must be using [Python>=3.8](https://github.com/fastapi/fastapi/blob/9606b916ef7c883f5ebeac1bf3db9adf5ae646a9/pyproject.toml#L10), but you are using Python version `3.6.8`. Upgrade your version of Python and everything should work as expected.

**Reply by Gelembjuk** [reply]
ok. makes sense. I have found some web page where it says python must be 3.6+ . This is why i tried at all.
For now i can not upgrade python. So, i can not consider fastapi for my project :(

Thank you

**Reply by Gelembjuk** [reply]
Maybe there is some "workaround" for python 3.6? 
To install manually, etc? clone repo and do something?

**Reply by sinisaos** [reply]
@Gelembjuk I'm afraid there is no other solution than to find some old version of `FastAPI` that used `Python 3.6` (there is no such version for FastAPI cli as it's a fairly new feature), but I don't recommend it due to security and lack of features.

**Reply by TriSi51** [reply]
You need to use this: pip install "fastapi[standard]". because it will download fastapi cli so that you can run the command fastapi.... Thanks for reading

**Reply by sinisaos** [reply]
@TriSi51 Sorry, but the question was how to run [FastAPI](https://github.com/fastapi/fastapi/blob/847296e885ed83bc333b6cc0a3000d6242083b87/pyproject.toml#L10) and [FastAPI cli](https://github.com/fastapi/fastapi-cli/blob/b65b8a48537a41faf66b5629f4f103356dced91b/pyproject.toml#L8) on **Python version 3.6**. I don't think it's possible because both require `Python>=3.8`.

**Reply by TriSi51** [reply]
> @TriSi51 Sorry, but the question was how to run [FastAPI](https://github.com/fastapi/fastapi/blob/847296e885ed83bc333b6cc0a3000d6242083b87/pyproject.toml#L10) and [FastAPI cli](https://github.com/fastapi/fastapi-cli/blob/b65b8a48537a41faf66b5629f4f103356dced91b/pyproject.toml#L8) on **Python version 3.6**. I don't think it's possible because both require `Python>=3.8`.
ah oke, I think we just need to upgrade to 3.8 since everyone now is using latest version of python

**Comment by Gelembjuk** [comment]
I have got now another env with python 3.11. So, all works fine.
3.6 is not a good version for FastAPI
